website/content/labs/lab3.md

```diff
@@ -2,22 +2,20 @@
 title: "Lab 3: Make"
 date: 2018-01-28
 publishdate: 2018-01-28
-draft: true
+draft: false
 ---
 
-**Due:** Thursday, April 25th, 2:30pm
+**Due:** Wednesday, April 29th, 8pm CDT
 
 In the class project, you will likely produce dozens of C files that will ultimately produce a single executable. When dealing with multiple source files, specially when there are dependencies between them, it is common to use a *build system* instead of manually compiling and linking all the files. in this lab, we’ll explore the ways a program can be “built”--that is, the way that source code is turned into binary code so that a computer can execute it. In some cases, “building” may refer to compiling a single file, but usually it refers to the whole process of linking and creating a project: which can include linking, compiling, and running tests. We’ll look at testing in later labs, and for this lab we’ll focus on the `make` command as a way to compile and build projects. 
 
 {{% warning %}}
-{{% md %}}
 **NOTE**: For some of these tasks, you may be tempted to look at the Makefiles included with libgeometry, and copy-paste parts of them into your Makefiles. There are two important reasons not to do this:
 
 1. It is important that you understand what you're doing in each of the tasks in this lab. If you get stuck and you're not sure how to proceed, ask us (either during the lab session or on Piazza). If you just copy-paste from one of our Makefiles, you won't understand how those parts of the Makefile work.
```

Comment:
Outdated reference to lab session and piazza

Reply:
Fixed!